platform/android/app/src/main/kotlin/coop/polypoly/polypod/homescreen/HomeScreenFragment.kt

```diff
@@ -120,7 +120,9 @@ fun createScreen(sectionModels: List<SectionModel>) {
     val bigTileWidth =
         containerWidth - smallTileWidth - containerLayout.horizontalInterItemSpacing.value // ktlint-disable max-line-length
 
-    val sections = sectionModels.map {
+    val sections = sectionModels
+        .filter { it.visible ?: true }
```

Comment:
I think the same should be done for Android as for iOS, filter out the categories in `importFeatures` function in FeatureStorage.

Reply:
Was just working on that, done :D

Reply:
:D Looking good now.